packages/cli/test/cli.spec.ts

```diff
@@ -268,6 +268,25 @@ describe('Stylable Cli', function () {
             expect(stdout, 'stdout').to.match(/unknown var "xxx"/);
         });
 
+        it.skip('(diagnosticsMode) should not exit with error when using strict mode with only info diagnostics', () => {
+            // Todo: test info diagnostic when we have one.
```

Comment:
it's fine to test with available diagnostics, but even then I would add an issue about aligning the options between webpack and CLI ( keeping a tech-debt to replace with a test that won't break once some info diagnostics is removed). Having the same hook `postProcess` hook or any other hook would allow to have the same test as webpack with a custom non temporary info diagnostics.

Reply:
I would love to have that feature, and we will probably add it once we know how to parse the stc config from the stylable config. (we can add it to the research)
but, I would never prefer a test that stubs info diagnostic than emitting real info diagnostic.
Until we have that use case, I will add a `todo` there that will be updated once we have permanent info diagnostic.